src/current/_includes/v25.3/sidebar-data/cloud-deployments.json

```diff
@@ -325,6 +325,12 @@
                         "urls": [
                             "/cockroachcloud/egress-perimeter-controls.html"
                         ]
+                    },
+                    {
+                    "title": "Egress Private Endpoints",
```

Comment:
Per convo with Devarshi, he doesn't feel this is appropriate for the specific topic of Network Authorization and should instead be under the higher-level Security topic